lfc-scheduler/manifests/permit/scheduler-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kubescheduler.config.k8s.io/v1beta2
```

Comment:
Could you already get this working with v1? -> https://kubernetes.io/docs/reference/scheduling/config/

Reply:
I tried and the scheduler failed at startup... I will give it an other shot otherwise leave it to beta2